cmd/synthetic-monitoring-agent/main.go

```diff
@@ -176,6 +171,16 @@ func run(args []string, stdout io.Writer) error {
 		zerolog.SetGlobalLevel(zerolog.ErrorLevel)
 	}
 
+	baseCtx, err := tracing.Context(context.Background())
+	if err != nil {
+		zl.Err(err).Msg("could not configure tracer provider, disabling tracing functionality")
+	}
+
+	baseCtx, cancel := context.WithCancel(baseCtx)
+	defer cancel()
```

Comment:
this is possibly nitpicking, but I would have expected this to come first.

Reply:
Makes sense, should be fixed now :)